artemis-server/src/main/java/org/apache/activemq/artemis/core/protocol/ProtocolName.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.activemq.artemis.core.protocol;
+
+import org.apache.activemq.artemis.api.core.SimpleString;
+
+public enum ProtocolName {
+
+   MQTT_PROTOCOL_NAME(SimpleString.toSimpleString("MQTT")),
```

Comment:
-1, Core should not know about specific protocols.

Reply:
Got it thanks.  Since the processing of cluster-notification is in the Core, and the Core does not depend on the specific protocol code. I will think about how to adjust the code structure and then submit a version. Maybe need a listener or a callback